examples/with-sfcc/pages/_document.js

```diff
@@ -0,0 +1,26 @@
+import Document, { Html, Head, Main, NextScript } from 'next/document'
+
+class MyDocument extends Document {
```

Comment:
Let's convert to a functional component: https://nextjs.org/docs/advanced-features/custom-document